documentation/book/con-kafka-upgrades-using-cluster-operator.adoc

```diff
@@ -11,6 +11,14 @@ How the Cluster Operator will perform an upgrade depends on the differences betw
 * The log message format version of the two Kafka versions
 * The version of Zookeeper used by the two Kafka versions
 
-When each of these versions is the same, as is typically the case for a patch level upgrade (for example 2.0.0 to 2.0.1), then the Cluster Operator can perform the upgrade using a single rolling update of the Kafka brokers. 
-When one or more of these versions differ, the Cluster Operator will require two or three rolling updates of the Kafka brokers to perform the upgrade.
+//Upstream only
+ifdef::StrimziUpgrades[]
```

Comment:
I guess it is fine to keep this Strimzi only - but you have to remember it since we will need to change it once/if we have support for 2.1.2 for example. So keeping it as an example would be easier.

Reply:
For the sake of simplicity, I've removed the example.